x/subscription/keeper/subscription_test.go

```diff
@@ -292,15 +292,35 @@ func TestRenewSubscription(t *testing.T) {
 	err = keeper.CreateSubscription(ts.ctx, creator, creator, ts.plans[0].Index, 12, "")
 	require.NotNil(t, err)
 
-	// but asking for additional 10 is fine
-	err = keeper.CreateSubscription(ts.ctx, creator, creator, ts.plans[0].Index, 10, "")
+	// but asking for additional 9 months (10 would also be fine)
```

Comment:
why this change?
the intent here was to ask for the max possible.

Reply:
on line 315, I try to extend the subscription for another month. I wanted to create a situation where the subscription extension would succeed if everything was ok but I still expect a failure since the plan changed (if I would've kept the 10 month extension, another 1 month extension would fail whether the plan changed or not)

Reply:
got it. so maybe append to the comment: ", but one more is coming below".